From 6ba8a3b4c99f55ee43a8c5c44424cab9c7350b6e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 12 Dec 2025 15:08:49 +0100
Subject: [PATCH 938/954] refactor(stm32mp2): print reset reason at a lower log
 level
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The reset reason is important information that should be printed even at
the NOTICE log level.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 783aeda2d2e498397765131d68130f0161624fee refactor(stm32mp2): print reset reason at a lower log level]
Change-Id: Ica071fcd92c6a0bf51d743bc7ac14ffaa9e6d998
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/517684
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 plat/st/stm32mp2/bl2_plat_setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/bl2_plat_setup.c b/plat/st/stm32mp2/bl2_plat_setup.c
index 0e1e8ed696..c60463f475 100644
--- a/plat/st/stm32mp2/bl2_plat_setup.c
+++ b/plat/st/stm32mp2/bl2_plat_setup.c
@@ -166,7 +166,7 @@ static void print_reset_reason(void)
 		}
 	}
 
-	INFO("Reset reason: %s (0x%x)\n", reason_str, rstsr);
+	NOTICE("Reset reason: %s (0x%x)\n", reason_str, rstsr);
 }
 
 #if STM32MP_M33_TDCID
-- 
2.53.0

